Fix SfxTabPage scope - force global namespace

This was a build breaker on macos
---

 sfx2/inc/sfx2/tabdlg.hxx |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)


diff --git sfx2/inc/sfx2/tabdlg.hxx sfx2/inc/sfx2/tabdlg.hxx
index 64186d6..4976abd 100644
--- sfx2/inc/sfx2/tabdlg.hxx
+++ sfx2/inc/sfx2/tabdlg.hxx
@@ -134,12 +134,12 @@ protected:
     virtual SfxItemSet*			CreateInputItemSet( USHORT nId );
     // wird *nicht* im Sfx gel"oscht!
     virtual const SfxItemSet*	GetRefreshedSet();
-    virtual void				PageCreated( USHORT nId, SfxTabPage &rPage );
+    virtual void				PageCreated( USHORT nId, ::SfxTabPage &rPage );
     virtual long                Notify( NotifyEvent& rNEvt );
 
     SfxItemSet*		pExampleSet;
     SfxItemSet*		GetInputSetImpl();
-    SfxTabPage*		GetTabPage( USHORT nPageId ) const;
+    ::SfxTabPage*	GetTabPage( USHORT nPageId ) const;
 
     BOOL			IsInOK() const;
     /**	prepare to leace the current page. Calls the DeactivatePage method of the current page, (if necessary),
